src/main/java/net/pms/dlna/protocolinfo/ProtocolInfoAttributeName.java

```diff
@@ -0,0 +1,256 @@
+
+			// Check for static instances
+			if (isBlank(name)) {
+				return NONE;
+			}
+			name = name.trim().toUpperCase(Locale.ROOT);
+
+			if ("*".equals(name)) {
+				return WILDCARD;
+			}
+
+			// Check for known instances
+			for (KnownProtocolInfoAttributeName knownAttribute : KnownProtocolInfoAttributeName.values()) {
+				if (name.equals(knownAttribute.getName())) {
+					return knownAttribute;
+				}
+			}
+
+			// Check for cached instances
+			instanceCacheLock.readLock().lock();
+			try {
+				for (ProtocolInfoAttributeName cachedAttribute : instanceCache) {
+					if (name.equals(cachedAttribute.getName())) {
+						return cachedAttribute;
+					}
+				}
+			} finally {
+				instanceCacheLock.readLock().unlock();
+			}
+			return null;
+		}
+
+		/**
+		 * Creates or retrieves a {@link ProtocolInfoAttributeName} instance
+		 * representing the attribute name {@code name}. Any existing predefined
+		 * or cached instances are first checked, and if none are found for
+		 * {@code value} a new instance is created.
+		 *
+		 * @param name the profile name.
+		 * @return The {@link ProtocolInfoAttributeName} instance.
+		 */
+		public ProtocolInfoAttributeName createAttributeName(String name) {
+
+			ProtocolInfoAttributeName instance = getAttributeName(name);
+			if (instance != null) {
+				return instance;
+			}
+
+			// Null values will already have been handled above
+			name = name.trim().toUpperCase(Locale.ROOT);
+
+			// Prepare to create a new instance
+			instanceCacheLock.writeLock().lock();
+			try {
+				// Check cache again as it could have been added while the lock
+				// was released
+				for (ProtocolInfoAttributeName cachedAttribute : instanceCache) {
+					if (name.equals(cachedAttribute.getName())) {
+						return cachedAttribute;
+					}
+				}
+
+				// None was found, create the instance
+				instance = new GenericProtocolInfoAttributeName(name);
+				instanceCache.add(instance);
+				LOGGER.trace("ProtocolInfoAttributeNameFactory added unknown attribute name \"{}\"", name);
+				return instance;
+			} finally {
+				instanceCacheLock.writeLock().unlock();
+			}
+		}
+	}
+
+	/**
+	 * This contains predefined, known {@code protocolInfo} attribute names.
+	 */
+	public enum KnownProtocolInfoAttributeName implements ProtocolInfoAttributeName {
+		// The order is important for protocolInfo
+
+		/** DLNA.ORG_PN */
+		DLNA_ORG_PN("DLNA.ORG_PN"),
+
+		/** ARIB.OR.JP_PN */
+		ARIB_OR_JP_PN("ARIB.OR.JP_PN"),
+
+		/** DTV_MVP_PN */
+		DTV_MVP_PN("DTV_MVP_PN"),
+
+		/** PANASONIC.COM_PN */
+		PANASONIC_COM_PN("PANASONIC.COM_PN"),
+
+		/** MICROSOFT.COM_PN */
+		MICROSOFT_COM_PN("MICROSOFT.COM_PN"),
+
+		/** SHARP.COM_PN */
+		SHARP_COM_PN("SHARP.COM_PN"),
+
+		/** SONY.COM_PN */
+		SONY_COM_PN("SONY.COM_PN"),
+
+		/** DLNA.ORG_OP */
+		DLNA_ORG_OP("DLNA.ORG_OP"),
+
+		/** DLNA.ORG_PS */
+		DLNA_ORG_PS("DLNA.ORG_PS"),
+
+		/** DLNA.ORG_CI */
+		DLNA_ORG_CI("DLNA.ORG_CI"),
+
+		/** DLNA.ORG_FLAGS */
```

Comment:
It's because i saw it many times while checking at your wireshark capture, and in the DLNA 2006 they stick all togother.